tests/manage/test_run_io_multiple_pods.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import logging
+from concurrent.futures import ThreadPoolExecutor
+
+from ocs_ci.framework.testlib import ManageTest, tier2
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_rbd_secret, create_pods, create_pvcs
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier2
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_ceph_block_pool.__name__,
+    create_rbd_storageclass.__name__,
+    create_pvcs.__name__,
+    create_pods.__name__
+)
+class TestRunIOMultiplePods(ManageTest):
+    """
+    Run IO on multiple pods in parallel
+    """
+    pvc_size = '5Gi'
+    num_of_pvcs = 50
+
+    def test_run_io_multiple_pods(self):
+        """
+        Run IO on multiple pods in parallel
+        """
+        results = list()
+
+        with ThreadPoolExecutor(max_workers=self.num_of_pvcs) as executor:
+            for pod in self.pod_objs:
+                results.append(executor.submit(pod.run_io('fs', '1G')))
+
+        for pod in self.pod_objs:
+            fio_result = pod.get_fio_results()
+            logger.info(f"IOPs after FIO for pod {pod.name}:")
+            logger.info(
+                f"Read: {fio_result.get('jobs')[0].get('read').get('iops')}"
+            )
+            logger.info(
+                f"Write: {fio_result.get('jobs')[0].get('write').get('iops')}"
+            )
```

Comment:
we need some validation.
From the execution output, I see that there are few pods that write/reads not in a ratio of 75%/25%.
may this is a bug?

Reply:
how accurate that should be? All the result look very close to 1/4 and 3/4